CHANGELOG_de-DE.md

```diff
@@ -1,3 +1,6 @@
+# 1.1.0
+* Wechsel auf das neue unzer SDK (https://packagist.org/packages/unzerdev/php-sdk)
+
 # 1.0.5
```

Comment:
this version does not exist and needs to be removed

Reply:
fixed